test-train/src/genius/Main.java

```diff
@@ -6,17 +6,36 @@
 public class Main {
 
     public static void main(String[] args) {
+        makeQuestion();
+    }
+
+    private static void calcMax() {
         Scanner scan = new Scanner(System.in);
```

Comment:
You can initialize Scanner in method main and send in calcMax as input parameter

Reply:
done